### DISCUSSION TITLE
Propuestas de mejora para versión 3.x

### DISCUSSION THREAD

**Discussion by eclipxe13** [phpcfdi/cfdi-sat-scraper#49] [General]
Estamos preparando la versión 3 del proyecto con soporte para entrar al sitio del SAT utilizando FIEL.
Este cambio provoca un cambio de compatibilidad con versiones previas, por lo que se necesita incrementar la versión mayor a 3.
Esto abre la oportunidad de mejorar y corregir la API (clases, métodos y nombres, etc.) para tener una mejor experiencia al usar la librería (Developer eXperience).

**¿Qué cosas cambiarías en esta librería para mejorar la experiencia de usuario?**

**Comment by carlossantillan778** [comment]
Hola @eclipxe13 , me interesa ver si me puedes ayudar a un tema de scraping con el sat e implementar la librería. ¿En dónde te puedo contactar?

**Reply by eclipxe13** [reply]
En la comunidad de phpCfdi de discord 
https://discord.gg/aFGYXvX
